src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -15,6 +15,7 @@
 package com.google.api.generator.engine.ast;
 
 import com.google.auto.value.AutoValue;
+import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
Can we avoid pulling in a new dependency and use Escaper from Guava instead? More dependencies will make version management difficult, as we've seen from the proto3 presence support. The range of special characters we should handle then (aside from common Java format characters) include the ones that are currently not correctly handled in the baseline file.

Reply:
ok, let me try the library out!

Reply:
tried to use Guava dependency and turns out the escape rules has to be defined manually like below one by one, same for UnicodeEscapter (more complicated). The direct Escapers.escape() method is non-static which can not be called from static context.
```
      private Escaper rmquote = Escapers.builder().addEscape(' " ', "\"").build();        
      private Escaper rmnewline = Escapers.builder().addEscape('\n', "\\\n").build();    
```
so it might worth to call a separate library to do the special handling for us I think, WDYT? related API summary is [here](https://guava.dev/releases/21.0/api/docs/com/google/common/escape/package-summary.html) Thanks

Reply:
Yes, the intent is to manually build the Escaper and the characters we need to escape. Some questions to consider that may help:
- If `escape()` cannot be called from a static context, how would you solve it?
- When does it make sense to build the Escaper?